roles/splunk_search_head/tasks/main.yml

```diff
@@ -20,13 +20,13 @@
 - include_tasks: peer_cluster_master.yml
   when:
     - splunk.indexer_cluster
-    - splunk.multisite_master is not defined
+    - splunk.set_search_peers is defined
```

Comment:
@kyanite If you're going to go this route, we need set_search_peers in addition to the multisite_master check - not replacing the multisite_master check

Reply:
Also, please change the check so that it can preserve backwards compatibility

Reply:
@kyanite Please make sure to update the tests as necessary for https://github.com/splunk/docker-splunk for this change to proceed. It will break the clustering test-scenarios

Reply:
Okay, I believe I was wrong about the clustering scenarios.  Approving